src/google-maps/google-map/google-map.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+  OnInit,
+} from '@angular/core';
+import {ReplaySubject} from 'rxjs';
+
+/**
+ * Angular component that renders a Google Map via the Google Maps JavaScript
+ * API.
+ * @see https://developers.google.com/maps/documentation/javascript/reference/
+ */
+@Component({
+  selector: 'google-map',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<div class="map-container"></div>',
+})
+export class GoogleMap implements OnInit {
+  // Arbitrarily chosen default size
+  @Input() height = '500px';
+  @Input() width = '500px';
+
+  // TODO(mbehrlich): add options, handlers, properties, and methods.
+
+  private readonly _map$ = new ReplaySubject<google.maps.Map>(1);
```

Comment:
Is this used for any functionality or testing at this time? It doesn't look like it.

Reply:
Not currently. My reasoning for creating this is a bit arbitrary, but I thought it would be fine for an initial cl. I will be using this subject later to handle child components and inputs, but for now it's just here to prevent an unused expression warning.